lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -66,9 +66,12 @@ class CourseGoalsSerializer(serializers.Serializer):
     """
     Serializer for Course Goal data
     """
-    goal_options = serializers.ListField()
+    goal_options = serializers.SerializerMethodField()
     selected_goal = serializers.DictField()
 
+    def get_goal_options(self, goals):
```

Comment:
this is returning goal_options: None for the new goals, but we can just remove this field altogether once we clean up the old goals

Reply:
It was a listfield before - was the empty state None or []? Might be nice to use `.get('goal_options', [])` to avoid tripping up old code - but maybe it's robust enough to avoid that issue.

Reply:
its only going to be None here if the flag for the new goals is on, but sure that's a good idea, [] makes more sense than None while we have this field around